app/Models/Traits/ResultDataAttributes.php

```diff
@@ -18,6 +18,16 @@ protected function downloadBits(): Attribute
         );
     }
 
+    /**
```

Comment:
You don't need to add `download_bytes` and `upload_bytes` as attributes, since they exist physically the model will load them.